backend/server.js

```diff
@@ -70,30 +89,42 @@ app.post('/', async (req, res) => {
   )
 
   // get faucet balance status, also remove addressese from array
-  const faucetStatus = await checkFaucetBalance(tokenAddresses, tokenAmounts)
+  const faucetStatus = await checkFaucetStatus(
+    tokenAddresses,
+    tokenAmounts,
+    checkSumAddress,
+  )
 
-  try {
-    const transaction = await faucetContract.methods
-      .sendMultiTokens(tokenAddresses, tokenAmounts, checkSumAddress)
-      .send({ gas: 9999999 })
+  if (tokenAddresses.length > 0) {
```

Comment:
if faucetStatus has err, what happened here?
I think you want to check faucetStatus first, right?

Reply:
Yes check faucetStatus first, **line 92**
inside `checkFaucetStatus`, if token has err, it is removed from `tokenAddresses` array.

So here **line 98**, length is 0 if all token have error

Reply:
oh, I see, it works, but it is a little confuse.